Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <PackageReference Include="DotNet.ReproducibleBuilds" Version="1.1.1" PrivateAssets="All"/>
```

Comment:
We already set most of the properties that DotNet.ReproducibleBuilds does. I think the only one that's missing is `    <DebugType>embedded</DebugType>` can we add just it?